### DISCUSSION TITLE
Can't add a second breed?

### DISCUSSION THREAD

**Discussion by TrishaGreenie** [sheltermanager/asm3#1429] [General]
Hi all,
I've been making use of the second breed feature when we occasionally get a cat that looks like a particular breed, but we don't have papers, and/or know it's a moggy with some genes of a particular breed. I can't seem to use this anymore. I am not sure if it's something I did? e.g. I recently added a couple of additional fields, including one in the Animal Details area where the Breed is selected. Would that have had any impact on the ability to add a second breed?

Thanks
Trisha

**Comment by Tetzor** [comment]
Hi,

We recently added a facility to specify which species have the second breed field available, by default it is only active for Dogs. You can turn the field on for Cats in Settings->Options->Add Animal.

Kind regards

Jon

**Comment by TrishaGreenie** [comment]
Hi
Thank you, I figured it out. For anyone else, I was confused by the wording on the options page as there is no check box at the bit where you select species - so you check the first box, "Show the breed fields", and then select Cat (or whatever species you want) from the "OR only show the second breed field for these species of animals" drop down - which has no check box.

Thanks
Trisha
